devices.ttl

```diff
@@ -683,11 +683,20 @@ _:boss-rc-10r device:typeA true .
 _:boss-rc-10r device:uri "https://www.boss.info/global/products/rc-10r" .
 
 _:retrokits-RK-006 device:make "Retrokits" .
-_:retrokits-RK-006 device:model "RK-006" .
+_:retrokits-RK-006 device:model "RK-006 Hub" .
 _:retrokits-RK-006 device:inputs 2 .
 _:retrokits-RK-006 device:outputs 10 .
 _:retrokits-RK-006 device:typeA true .
 _:retrokits-RK-006 device:uri "https://www.retrokits.com/shop/rk006" .
+_:retrokits-RK-006 device:details "TRS-A output tip switchable to Gate/PWM".
+
+_:retrokits-RK-008 device:make "Retrokits" .
+_:retrokits-RK-008 device:model "RK-008" .
+_:retrokits-RK-008 device:inputs 2 .
+_:retrokits-RK-008 device:outputs 2 .
+_:retrokits-RK-008 device:typeA true .
+_:retrokits-RK-008 device:uri "https://www.retrokits.com/shop/rk008" .
+_:retrokits-RK-008 device:details "New MIDI Device".
```

Comment:
Has this device been officially released yet? Once the page referenced in `uri` is up, and if you could add more descriptive short-form `notes` and/or long-form `details`, I'd be happy to merge this.